code/z_adventurezones/lavamoon.dm

```diff
@@ -1110,29 +1116,29 @@ var/global/iomoon_blowout_state = 0 //0: Hasn't occurred, 1: Moon is irradiated
 
 				return 0
 
-
-	rotor
-		name = "giant rotors"
+	base
+		name = "huge contraption"
 		desc = "An enormous artifact of some sort. You feel uncomfortable just being near it. Probably because it is a giant piece of dangerous machinery."
+		anchored = ANCHORED
+		density = 0
 		icon = 'icons/effects/160x160.dmi'
-		icon_state = "powercore_rotors_off"
+		icon_state = "powercore_base"
 		bound_height = 160
 		bound_width = 160
-		layer = 3.9 // TODO layer
-		density = 0
+		layer = 3.2 // TODO layer
 
 		var/obj/iomoon_boss/core/core = null
 
-	base
-		name = "huge contraption"
-		desc = "An enormous artifact of some sort. You feel uncomfortable just being near it."
+	top
```

Comment:
Top is causing issues with click detection, it's layered over the current core.
It seems to only exist to play the startup animation? That should be done on core and this removed.

Reply:
Top shouldn't be causing any click detection issues-- its layer 7, whereas the core is layer 10.
top is the object with the actual mechanism core sprite, that weird glowing light pillar and such
It's seperate from core to save a headache or two with the way attacking the core seems to be intended to work & the way large sprites place themselves
The click detection issue was a seperate thing- I've tweaked the core some, it shouldn't be as finnicky now